api/services/kakaoService.js

```diff
@@ -0,0 +1,29 @@
+const axios = require('axios')
+
+const getAccessTokenByKakao = async(code, res) => {
```

Comment:
```suggestion
const getAccessTokenByKakao = async(code) => {
```
res는 parameter로 받을 필요가 있을까요?